NugetSpecs/Microsoft.Dynamics365.UIAutomation.Api.nuspec

```diff
@@ -2,27 +2,29 @@
 <package >
   <metadata>
     <id>Dynamics365.UIAutomation.Api</id>
-    <version>9.0.0.1</version>
-    <title>Automated UI testing API for •Microsoft Dynamics 365 Online Version 9.0 (9.0.x) (DB 9.0.x) online</title>
-    <authors>Ranger Team</authors>
+    <version>9.2.21111.00116-RW2-Preview</version>
+    <title>Automated UI testing API for •Microsoft Dynamics 365 Online Version 9.2 2021 Release Wave 2 beta (9.2.21111.00116) (DB 9.2.21111.00116)</title>
+    <authors>PowerApps Lifecycle Quality Team</authors>
```

Comment:
Recommend changing this to 'Power Apps' or 'Power Apps Studio' as those are less likely to change over time.